modules/support_ticket/config/install/views.view.users_support_tickets.yml

```diff
@@ -0,0 +1,844 @@
+            operator: field_state_value_op
+            identifier: field_state_value
+            required: false
+            remember: true
+            multiple: true
+            remember_roles:
+              anonymous: '0'
+              authenticated: '0'
+              administrator: '0'
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          reduce_duplicates: false
+          plugin_id: list_field
+        field_priority_value:
+          id: field_priority_value
+          table: support_ticket__field_priority
+          field: field_priority_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: or
+          value:
+            lowest: lowest
+            low: low
+            normal: normal
+            high: high
+            urgent: urgent
+            critical: critical
+          group: 1
+          exposed: true
+          expose:
+            operator_id: field_priority_value_op
+            label: Priority
+            description: ''
+            use_operator: false
+            operator: field_priority_value_op
+            identifier: field_priority_value
+            required: false
+            remember: true
+            multiple: true
+            remember_roles:
+              anonymous: '0'
+              authenticated: '0'
+              administrator: '0'
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+          reduce_duplicates: false
+          plugin_id: list_field
+      sorts: {  }
+      title: 'User''s support tickets'
+      header: {  }
+      footer: {  }
+      empty:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: true
+          tokenize: false
+          content:
+            value: 'There are currently no support tickets to display.'
+            format: basic_html
+          plugin_id: text
+      relationships:
+        field_assigned_to:
+          id: field_assigned_to
+          table: support_ticket__field_assigned_to
+          field: field_assigned_to
+          relationship: none
+          group_type: group
+          admin_label: 'field_assigned_to: User'
+          required: true
+          plugin_id: standard
+      arguments:
```

Comment:
Should this be a filter on the current user rather than an argument?

Or generally not sure if the idea is to be able to explicitly see other people's tickets or not - so depends on that I guess.